Changelog.md

```diff
@@ -2,6 +2,17 @@
 
 This file only reflects the changes that are made in this image. Please refer to the upstream GitLab [CHANGELOG](https://gitlab.com/gitlab-org/gitlab-foss/blob/master/CHANGELOG.md) for the list of changes in GitLab.
 
+**14.9.0**
+- gitlab: upgrade CE to v14.9.0 - https://about.gitlab.com/releases/2022/03/22/gitlab-14-9-released/
+- gitaly: upgrade to v14.9.0
+- gitlab-pages: upgrade to v1.56.0
+- gitlab-shell: upgrade to v13.24.1
```

Comment:
The version seems to be inkonsistent with https://gitlab.com/gitlab-org/gitlab-foss/-/blob/v14.9.0/GITLAB_SHELL_VERSION .

Reply:
thanks a lot for the catch and PR update @sachilles, noted for next time!